### DISCUSSION TITLE
Why to disable treesitter's latex?

### DISCUSSION THREAD

**Discussion by ChenZhongPu** [LazyVim/LazyVim#1242] [Q&A]
I noticed that the extra `lang.tex` would disable treesitter's latex and prefer the highlight from `vimtex`. But in fact, I think there is no need to disable it because the highlights provided by treesitter are better.

Rendered by treesitter:

<img width="836" alt="image" src="https://github.com/LazyVim/LazyVim/assets/2310958/b2e6d71a-b007-443d-bfde-9c179ef489fe">

Rendered by vimtex:

<img width="771" alt="image" src="https://github.com/LazyVim/LazyVim/assets/2310958/d4ea7c92-b26e-45f9-9f56-5921b71b0a5e">

**Comment by jyuan0** [comment]
Not sure it helps, but it was discussed a bit in the PR: https://github.com/LazyVim/LazyVim/pull/1156
